src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/project.json

```diff
@@ -1,5 +1,11 @@
 {
-  "description": "ASP.NET Identity providers that use Entity Framework.",
+  "description": "ASP.NET Core Identity providers that use Entity Framework Core.",
```

Comment:
I'll keep messing it up until the SHA hash spells out a word that expresses how silly I feel :smile: